{"name": "test_fixture_zcc2[cc_calculator_shuju2]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1.0 == 2000\n  +1.0\n  -2000", "trace": "self = <cc_work.test_fixture_zcc.TestCal_cc_new object at 0x00000000039B46D8>, cc_calculator_func = <cc_work.calculator_cc.Calculator object at 0x00000000039DAF28>, cc_calculator_shuju = [1000, 1000, 2000]\n\n    @pytest.mark.run(order=4)\n    def test_fixture_zcc2(self, cc_calculator_func, cc_calculator_shuju):\n        result = cc_calculator_func.div(cc_calculator_shuju[0], cc_calculator_shuju[1])\n        print('这是测试用例2')\n        # print(result)\n>       assert result == cc_calculator_shuju[2]\nE       assert 1.0 == 2000\nE         +1.0\nE         -2000\n\ntest_fixture_zcc.py:32: AssertionError"}, "parameters": [{"name": "cc_calculator_shuju", "value": "[1000, 1000, 2000]"}], "start": 1609398358119, "stop": 1609398358119, "uuid": "8c06d1a3-c989-4484-9cd1-144ade13413e", "historyId": "3fc11c164d222253aadd40f92a5371c5", "testCaseId": "6207987451ade26d46e5eac58d4c08e9", "fullName": "test_fixture_zcc.TestCal_cc_new#test_fixture_zcc2", "labels": [{"name": "suite", "value": "test_fixture_zcc"}, {"name": "subSuite", "value": "TestCal_cc_new"}, {"name": "host", "value": "admin-PC"}, {"name": "thread", "value": "12680-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_fixture_zcc"}]}